frontend/src/pages/MyCourses/Course.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import Image from 'components/Image';
+import CourseInfo from 'components/CourseInfo';
+
+import {
+  CourseContainer,
+  ImageWrapper,
+  CourseTitle,
+  CourseDescription,
+  CourseButton,
+  ButtonsContainer,
+  Divider,
+} from './styled';
+
+interface Props {
+  title: string;
+  description: string;
+  language: string;
+  duration: string;
+  lessons: string;
+  link: string;
+}
+
+const CourseItem: React.FC<Props> = ({ title, description, language, duration, lessons, link }) => (
```

Comment:
Looks like common component, but check it please with Irina

Reply:
It is almost the same on all pages. The only difference is its size and buttons. Should I create common component for all pages that receives size and buttons?